src/main/default/classes/ExceptionManager.cls

```diff
@@ -0,0 +1,57 @@
+public class ExceptionManager {
+    public void manage(Exception e, TriggerContext context, HandlerConfig config, Savepoint sp) {
+        System.debug(e);
+        switch on config.onException {
+            when SUPPRESS {
+                System.debug('Exception Suppress!');
+                Database.rollback(sp);
+            }
+            when EMAIL {
+                try {
+                    sendEmail(e, context, config);
```

Comment:
Actually Sending the Email here doesn't really work as the next exception could "throw" and cause it to be rolled back.  Will need to store the data in a static object and send at the end

Reply:
eh... that doesn't really work. Something could always throw and roll back emails.

I think we need a platform event to actually escape the transaction